great-idea/css/index.css

```diff
@@ -63,4 +63,87 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+.container {
+	width: 880px;
+	margin: 0 auto;
+}
+
+.navbar {
+	border: 1px solid red;
```

Comment:
i think this border is here because maybe you're still working on it? you probably want to toss some flex properties on there so that you can align the a tags and the logo easily.